pyproject.toml

```diff
@@ -777,4 +777,18 @@ constraint-dependencies = [
     "stack_data>=0.6.0",
     "traitlets>=5.13.0",
     "typing_extensions>=4.6; python_version<'3.12'",
+
+    # via ipython (can be removed once ipython<8.17 is dropped)
+    "pickleshare>=0.7.5",
+
+    # via beautifulsoup4
+    "soupsieve>=1.6.1", # https://code.launchpad.net/~neutrinoceros/beautifulsoup/+git/beautifulsoup/+merge/495860
```

Comment:
Upstream has merged your patch, so the comment should state the last version of the dependency that requires the constraint (both for `beautifulsoup4` and for `dask`).